ZEngine/include/ZEngine/Rendering/Specifications/FormatSpecification.h

```diff
@@ -112,4 +112,79 @@ namespace ZEngine::Rendering::Specifications
         COMPUTE
     };
 
+    enum class ImageViewType : uint32_t
+    {
+        TYPE_1D = 0 , 
+        TYPE_2D, 
+        TYPE_3D,
+        TYPE_CUBE
+    };
+
+    static VkImageViewType ImageViewTypeMap[] = {
+        VK_IMAGE_VIEW_TYPE_1D,
+        VK_IMAGE_VIEW_TYPE_2D,
+        VK_IMAGE_VIEW_TYPE_3D,
+        VK_IMAGE_VIEW_TYPE_CUBE,
+        VK_IMAGE_VIEW_TYPE_1D_ARRAY,
+        VK_IMAGE_VIEW_TYPE_2D_ARRAY,
+        VK_IMAGE_VIEW_TYPE_CUBE_ARRAY,
+        VK_IMAGE_VIEW_TYPE_MAX_ENUM };
+
+    enum class ImageCreateFlag {
+        SPARSE_BINDING_BIT = 0,
```

Comment:
sparse_binding_bit is actually 1  form vulkan header,
so for 0 you may want to introduce a default value .... maybe NONE = 0 ?